test/lib/range_parser_test.rb

```diff
@@ -33,4 +33,26 @@ class RangeParserTest < ActiveSupport::TestCase
     assert_equal (0.1..0.33), RangeParser.parse('0.1   ,   0.33')
   end
 
+  test 'parses garbage format from JSONAPI' do
```

Comment:
Having the brackets allows me to simply `JSON.parse` the value of the param, which is kinda nice. It's at least technically a valid JSON data type.